src/explorer/ExplorationLauncher.cs

```diff
@@ -17,47 +19,73 @@ public ExplorationLauncher(IContainer rootContainer)
         }
 
         /// <summary>
-        /// Configure an exploration within a given scope.
+        /// Configure a column exploration within a given scope.
         /// </summary>
         /// <param name="scope">The scoped container to use for object resolution.</param>
-        /// <param name="ctx">An <see cref="ExplorerContext" /> defining the exploration parameters.</param>
         /// <param name="conn">A DConnection configured for the Api backend.</param>
+        /// <param name="ctx">An <see cref="ExplorerContext" /> defining the exploration parameters.</param>
         /// <param name="componentConfiguration">
         /// An action to add and configure the components to use in this exploration.
         /// </param>
-        /// <returns>The running Task.</returns>
-        public static Exploration Explore(
+        /// <returns>A new ColumnExploration object.</returns>
+        public static ColumnExploration ExploreColumn(
             INestedContainer scope,
-            ExplorerContext ctx,
             DConnection conn,
+            ExplorerContext ctx,
             Action<ExplorationConfig> componentConfiguration)
         {
             // Configure a new Exploration
-            return Exploration.Configure(scope, _ =>
-            {
-                _.UseConnection(conn);
-                _.UseContext(ctx);
-                _.Compose(componentConfiguration);
-            });
+            var config = new ExplorationConfig(scope);
+            config.UseConnection(conn);
+            config.UseContext(ctx);
+            config.Compose(componentConfiguration);
+            return new ColumnExploration(config, scope.GetInstance<MetricsPublisher>(), ctx.Column);
```

Comment:
It seems that the `scope` is still disposed automatically on line 60, in the `LaunchColumnExploration` method, but this means that the lifetime of the `scope` is not the same as the lifetime of the `ColumnExploration`; this makes it difficult to understand why tests are passing without problems. (I also made an additional change so `LaunchExploration` will actually call `LaunchColumnExploration`.)

Reply:
Yes it was being disposed in `LaunchColumnExploration` but like you said, this is before the exploration ends (and `LaunchColumnExploration` was not being called in any case).

I think it's not causing a problem at the moment because we only use the `scope` to resolve our instances and once they are created, their lifetimes are managed by the runtime, so even if `scope` is disposed they are tracked by the garbage collector. 

However, if we don't dispose the scope at all, we will definitely get a memory leak. If we dispose the scope early (as we currently do) it should be fine as long as (a) we only use the `scope` to instantiate our components and then never touch it again and (b) none of the components that are tied to the scope are `IDisposable`, since in this case they will be disposed along with the `scope`.

I think the safest way to ensure correct behaviour is to tie the lifetime of the `scope` to the lifetime of the `Exploration`.

Reply:
I created a separated issue for this problem #173.